apps/E2E/src/LinkExperimental/specs/ExperimentalLink.spec.win.ts

```diff
@@ -21,6 +21,41 @@ describe('Link Testing Initialization', function () {
   });
 });
 
+describe('Link Testing Functionality', function () {
+  /* Scrolls and waits for the Button to be visible on the Test Page */
+  beforeEach(async () => {
+    await ExperimentalLinkPageObject.scrollToTestElement();
+  });
+
+  it('Validate OnPress() callback was fired on a click', async () => {
+    await ExperimentalLinkPageObject.click(ExperimentalLinkPageObject._secondaryComponent);
+    await expect(await ExperimentalLinkPageObject.didOnClickCallbackFire()).toBeTruthy(
+      `The link failed to fire an onClick callback with a mouse click.`,
+    );
+    await expect(await ExperimentalLinkPageObject.didAssertPopup()).toBeFalsy(ExperimentalLinkPageObject.ERRORMESSAGE_ASSERT);
+
+    await ExperimentalLinkPageObject.click(ExperimentalLinkPageObject._secondaryComponent); // Reset Button State
```

Comment:
I don't think we need this? Are we resetting the Link state?

Reply:
It might've been copied from a different test.